### PR TITLE
add nationality on adherent entity

### PR DESCRIPTION
**Pull request by Raulnet**

### CODE REVIEW

File: src/Entity/Adherent.php [hunk 1]
```diff
@@ -1412,4 +1420,9 @@ public function setDisplayMedia(bool $displayMedia): void
     {
         $this->displayMedia = $displayMedia;
     }
+
+    public function getNationality(): ?string
```
Comment (ottaviano) [inline src/Entity/Adherent.php]: what about a setter?

File: src/Membership/MembershipRequest.php [hunk 2]
```diff
@@ -91,6 +91,12 @@ class MembershipRequest implements MembershipInterface
      */
     public $conditions;
 
+    /**
```
Comment (ottaviano) [inline src/Membership/MembershipRequest.php]: `@var string|null`

File: templates/adherent/profile.html.twig [hunk 3]
```diff
@@ -82,6 +82,13 @@
         <div class="space--10-0"></div>
 
         <div class="l__row l__row--top l__row--h-stretch l__mobile--col">
+
+            <div class="l__row l__row--h-stretch b__nudge--bottom-tiny form__radio-button">
+                {{ form_widget(form.nationality, { attr: { class: 'register__form__profession' } }) }}
```
Comment (ottaviano) [inline templates/adherent/profile.html.twig]: why do you need `register__form__profession` class?

File: templates/adherent/profile.html.twig [hunk 4]
```diff
@@ -82,6 +82,13 @@
         <div class="space--10-0"></div>
 
         <div class="l__row l__row--top l__row--h-stretch l__mobile--col">
+
+            <div class="l__row l__row--h-stretch b__nudge--bottom-tiny form__radio-button">
```
Comment (ottaviano) [inline templates/adherent/profile.html.twig]: why do you need `form__radio-button`? it's not a radio, but select element, no?

File: templates/membership/join.html.twig [hunk 5]
```diff
@@ -98,6 +98,10 @@
                                 {{ form_errors(form.password) }}
                             </div>
                         {% endif %}
+                        <div class="l__row l__row--h-stretch register__form--full">
+                            {{ form_errors(form.nationality) }}
```
Comment (ottaviano) [inline templates/membership/join.html.twig]: in this template, the order of block is: `widget, errors`
but you display errors first and widget after. there is a real reason for that?

File: translations/validators.fr.yml [hunk 6]
```diff
@@ -29,6 +29,9 @@ common.email.repeated: Les adresses email ne correspondent pas.
 common.country.not_blank: Cette valeur ne doit pas être vide.
 common.country.invalid: Ce pays n'est pas valide.
 
+common.nationality.not_blank: Cette valeur ne doit pas être vide.
```
Comment (ottaviano) [inline translations/validators.fr.yml]: Why do you need to custom the default NotBlank constrain's message?

File: src/Entity/Adherent.php [hunk 7]
```diff
@@ -393,7 +398,8 @@ public static function create(
         string $registeredAt = 'now',
         ?array $tags = [],
         ?array $referentTags = [],
-        ?array $mandates = null
+        ?array $mandates = null,
```
[the remark this answers is not in the document]
Comment (Raulnet) [reply src/Entity/Adherent.php]: edited

File: src/Entity/Adherent.php (hunk 1 above)
Comment (Raulnet) [reply src/Entity/Adherent.php]: not need in this case

File: src/Membership/MembershipRequest.php (hunk 2 above)
Comment (Raulnet) [reply src/Membership/MembershipRequest.php]: added

File: templates/adherent/profile.html.twig (hunk 3 above)
Comment (Raulnet) [reply templates/adherent/profile.html.twig]: edited

File: templates/adherent/profile.html.twig (hunk 4 above)
Comment (Raulnet) [reply templates/adherent/profile.html.twig]: removed

File: translations/validators.fr.yml (hunk 6 above)
Comment (Raulnet) [reply translations/validators.fr.yml]: All NotBlank's messages are override on object MembershipRequest, I think it's for the translation

File: templates/membership/join.html.twig (hunk 5 above)
Comment (Raulnet) [reply templates/membership/join.html.twig]: reversed

File: templates/membership/register.html.twig [hunk 8]
```diff
@@ -60,6 +60,12 @@
                     {{ form_widget(form.password, {attr: {placeholder: 'Mot de passe', autocomplete: "new-password"} }) }}
                 </li>
 
+                <li>
+                    {{ form_label(form.nationality,'Nationalité',{ label_attr: {class: 'visually-hidden'} }) }}
```
[the remark this answers is not in the document]
Comment (Raulnet) [reply templates/membership/register.html.twig]: all file is fixed

**Pull request merged**